_rules/table-header-cells-refer-to-data-cells-d0f69e.md

```diff
@@ -0,0 +1,188 @@
+
+- [cell](https://www.w3.org/TR/wai-aria-1.1/#cell)
+- [gridcell](https://www.w3.org/TR/wai-aria-1.1/#gridcell)
+- [rowheader](https://www.w3.org/TR/html-aria/#index-aria-rowheader)
+- [columnheader](https://www.w3.org/TR/html-aria/#index-aria-columnheader)
+
+## Expectation
+
+Each target element is either the row or column header for cells that is a [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) of the same element as the target element.
+
+## Assumptions
+
+_There are currently no assumptions._
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 1.3.1: Information and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [H43: Using id and headers attributes to associate data cells with header cells in data tables](https://www.w3.org/WAI/WCAG21/Techniques/html/H43)
+- [Forming relationships between data cells and header cells](https://html.spec.whatwg.org/multipage/tables.html#header-and-data-cell-semantics)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+Each column header element has corresponding cells, within the same table element.
+
+```html
+<table>
+	<tr>
+		<th>Time</th>
+	</tr>
+	<tr>
+		<td>05:41</td>
+	</tr>
+</table>
+```
+
+#### Passed Example 2
+
+Each column header element has corresponding cells, within the same element having a [semantic role](#semantic-role) of `table`.
+
+```html
+<div role="table">
+	<div role="rowgroup">
+		<div role="row">
+			<span role="columnheader">Month</span>
+			<span role="columnheader">Top Temperature</span>
+		</div>
+	</div>
+	<div role="rowgroup">
+		<div role="row">
+			<span role="cell">July</span>
+			<span role="cell">40 C</span>
+		</div>
+		<div role="row">
+			<span role="cell">August</span>
+			<span role="cell">45 C</span>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed Example 3
+
+Each column header element has corresponding cells within the same table element. In this example the column headers have cells that span multiple columns.
+
+```html
+<table>
+	<thead>
+		<tr>
+			<th>Projects</th>
+			<th>Exams</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td colspan="2">15%</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Passed Example 4
+
+Each row and column header element has corresponding cells, within the same element having a [semantic role](#semantic-role) of `grid`.
+
+```html
+<table role="grid">
+	<thead role="rowgroup">
+		<tr role="row">
+			<td></td>
+			<th role="columnheader">Breakfast</th>
+			<th role="columnheader">Lunch</th>
+			<th role="columnheader">Dinner</th>
+		</tr>
+	</thead>
+	<tbody role="rowgroup">
+		<tr role="row">
+			<th scope="row" role="rowheader">Day 1</th>
+			<td>8:00</td>
+			<td>13:00</td>
+			<td>18:00</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+### Failed
+
+#### Failed Example 1
+
+Not all column headers have corresponding cells within the same table element.
```

Comment:
"Not all" => point out which one is failing. Technically, this example has 2 test targets and one is passing while the other fails.